### PR TITLE
chore: Refactor nasenvelopenet

### PR DESCRIPTION
**Pull request by gaocegege**
nasenvelopenet does not follow our convention. This PR is to refactor it.

Signed-off-by: Ce Gao <gaoce@caicloud.io>

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/492)

### CODE REVIEW

File: test/scripts/v1alpha1/run-tests.sh [hunk 1]
```diff
@@ -71,11 +71,15 @@ sed -i -e "s@image: katib\/vizier-core-rest@image: ${REGISTRY}\/${REPO_NAME}\/vi
...
+
 sed -i -e "s@image: katib\/suggestion-random@image: ${REGISTRY}\/${REPO_NAME}\/suggestion-random:${VERSION}@" manifests/v1alpha1/vizier/suggestion/random/deployment.yaml
 sed -i -e "s@image: katib\/suggestion-grid@image: ${REGISTRY}\/${REPO_NAME}\/suggestion-grid:${VERSION}@" manifests/v1alpha1/vizier/suggestion/grid/deployment.yaml
 sed -i -e "s@image: katib\/suggestion-hyperband@image: ${REGISTRY}\/${REPO_NAME}\/suggestion-hyperband:${VERSION}@" manifests/v1alpha1/vizier/suggestion/hyperband/deployment.yaml
 sed -i -e "s@image: katib\/suggestion-bayesianoptimization@image: ${REGISTRY}\/${REPO_NAME}\/suggestion-bayesianoptimization:${VERSION}@" manifests/v1alpha1/vizier/suggestion/bayesianoptimization/deployment.yaml
 sed -i -e "s@image: katib\/suggestion-nasrl@image: ${REGISTRY}\/${REPO_NAME}\/suggestion-nasrl:${VERSION}@" manifests/v1alpha1/vizier/suggestion/nasrl/deployment.yaml
+sed -i -e "s@image: katib\/suggestion-nasrl@image: ${REGISTRY}\/${REPO_NAME}\/suggestion-nasrl:${VERSION}@" manifests/v1alpha1/vizier/suggestion/nasrl/deployment.yaml
```
Comment (andreyvelich) [inline test/scripts/v1alpha1/run-tests.sh]: Do we need this line?

**Comment by hougangliu** [comment]
@gaocegege #494 duplicate with this PR, please enable nasenvelopenet test step in test/workflows/components/workflows-v1alpha1.libsonnet.
Thanks

File: test/scripts/v1alpha1/run-tests.sh (hunk 1 above)
Comment (gaocegege) [reply test/scripts/v1alpha1/run-tests.sh]: Fixed, it is a bug.

File: test/scripts/v1alpha1/run-tests.sh [hunk 2]
```diff
@@ -71,11 +71,14 @@ sed -i -e "s@image: katib\/vizier-core-rest@image: ${REGISTRY}\/${REPO_NAME}\/vi
...
+
 sed -i -e "s@image: katib\/suggestion-random@image: ${REGISTRY}\/${REPO_NAME}\/suggestion-random:${VERSION}@" manifests/v1alpha1/vizier/suggestion/random/deployment.yaml
 sed -i -e "s@image: katib\/suggestion-grid@image: ${REGISTRY}\/${REPO_NAME}\/suggestion-grid:${VERSION}@" manifests/v1alpha1/vizier/suggestion/grid/deployment.yaml
 sed -i -e "s@image: katib\/suggestion-hyperband@image: ${REGISTRY}\/${REPO_NAME}\/suggestion-hyperband:${VERSION}@" manifests/v1alpha1/vizier/suggestion/hyperband/deployment.yaml
 sed -i -e "s@image: katib\/suggestion-bayesianoptimization@image: ${REGISTRY}\/${REPO_NAME}\/suggestion-bayesianoptimization:${VERSION}@" manifests/v1alpha1/vizier/suggestion/bayesianoptimization/deployment.yaml
 sed -i -e "s@image: katib\/suggestion-nasrl@image: ${REGISTRY}\/${REPO_NAME}\/suggestion-nasrl:${VERSION}@" manifests/v1alpha1/vizier/suggestion/nasrl/deployment.yaml
+sed -i -e "s@image: katib\/suggestion-nasrl@image: ${REGISTRY}\/${REPO_NAME}\/suggestion-nasenvelopenet:${VERSION}@" manifests/v1alpha1/vizier/suggestion/nasenvelopenet/deployment.yaml
```
Comment (hougangliu) [inline test/scripts/v1alpha1/run-tests.sh]: katib\/suggestion-nasrl. -> katib\/suggestion-nasenvelopenet

**Comment by hougangliu** [comment]
> @hougangliu Thanks, added. Not aware of your PR, sorry about it.

Never mind, it seems we fix it at same time :)

File: test/scripts/v1alpha1/run-tests.sh (hunk 2 above)
Comment (gaocegege) [reply test/scripts/v1alpha1/run-tests.sh]: Fixed, thanks.

File: cmd/suggestion/nasenvelopenet/Dockerfile [hunk 3]
```diff
@@ -1,8 +1,8 @@
 FROM python:3.6
 
 ADD . /usr/src/app/github.com/kubeflow/katib
-WORKDIR /usr/src/app/github.com/kubeflow/katib/cmd/suggestion/nasenvelopenet
+WORKDIR /usr/src/app/github.com/kubeflow/katib/cmd/suggestion/nasenvelopenet/v1alpha1
 RUN pip install --no-cache-dir -r requirements.txt
-ENV PYTHONPATH /usr/src/app/github.com/kubeflow/katib:/usr/src/app/github.com/kubeflow/katib/pkg/api/python
+ENV PYTHONPATH /usr/src/app/github.com/kubeflow/katib:/usr/src/app/github.com/kubeflow/katib/pkg/api/v1alpha1/python
```
Comment (hougangliu) [inline cmd/suggestion/nasenvelopenet/Dockerfile]: RUN pip install --no-cache-dir -r ../requirements.txt

Comment (gaocegege) [reply cmd/suggestion/nasenvelopenet/Dockerfile]: Really thanks for your review, updated.

File: scripts/v1alpha1/build.sh [hunk 4]
```diff
@@ -40,10 +40,10 @@ docker build -t ${PREFIX}/v1alpha1/suggestion-grid -f ${CMD_PREFIX}/suggestion/g
 docker build -t ${PREFIX}/v1alpha1/suggestion-hyperband -f ${CMD_PREFIX}/suggestion/hyperband/v1alpha1/Dockerfile .
 docker build -t ${PREFIX}/v1alpha1/suggestion-bayesianoptimization -f ${CMD_PREFIX}/suggestion/bayesianoptimization/v1alpha1/Dockerfile .
 docker build -t ${PREFIX}/v1alpha1/suggestion-nasrl -f ${CMD_PREFIX}/suggestion/nasrl/v1alpha1/Dockerfile .
+docker build -t ${PREFIX}/suggestion-nasenvelopenet -f ${CMD_PREFIX}/suggestion/nasenvelopenet/v1alpha1/Dockerfile .
```
Comment (andreyvelich) [inline scripts/v1alpha1/build.sh]: Should it be `docker build -t ${PREFIX}/v1alpha1/suggestion-nasenvelopenet` here?

File: test/scripts/v1alpha1/build-suggestion-nasenvelopenet.sh [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+mkdir -p ${GO_DIR}
+cp -r cmd ${GO_DIR}/cmd
+cp -r pkg ${GO_DIR}/pkg
+cp -r vendor ${GO_DIR}/vendor
+
+cd ${GO_DIR}
+
+cp cmd/suggestion/nasenvelopenet/Dockerfile .
+gcloud builds submit . --tag=${REGISTRY}/${REPO_NAME}/suggestion-nasenvelopenet:${VERSION} --project=${PROJECT}
+gcloud container images add-tag --quiet ${REGISTRY}/${REPO_NAME}/suggestion-nasenvelopenet:${VERSION} ${REGISTRY}/${REPO_NAME}/suggestion-nasenvelopenet:latest --verbosity=info
```
Comment (andreyvelich) [inline test/scripts/v1alpha1/build-suggestion-nasenvelopenet.sh]: I think, here as well.
```
cp cmd/suggestion/v1alpha1/nasenvelopenet/Dockerfile .
gcloud builds submit . --tag=${REGISTRY}/${REPO_NAME}/v1alpha1/suggestion-nasenvelopenet:${VERSION} --project=${PROJECT}
gcloud container images add-tag --quiet ${REGISTRY}/${REPO_NAME}/v1alpha1/suggestion-nasenvelopenet:${VERSION} ${REGISTRY}/${REPO_NAME}/suggestion-nasenvelopenet:latest --verbosity=info

```

File: test/scripts/v1alpha2/build-suggestion-nasenvelopenet.sh [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+mkdir -p ${GO_DIR}
+cp -r cmd ${GO_DIR}/cmd
+cp -r pkg ${GO_DIR}/pkg
+cp -r vendor ${GO_DIR}/vendor
+
+cd ${GO_DIR}
+
+cp cmd/suggestion/nasenvelopenet/Dockerfile .
+gcloud builds submit . --tag=${REGISTRY}/${REPO_NAME}/suggestion-nasenvelopenet:${VERSION} --project=${PROJECT}
+gcloud container images add-tag --quiet ${REGISTRY}/${REPO_NAME}/suggestion-nasenvelopenet:${VERSION} ${REGISTRY}/${REPO_NAME}/suggestion-nasenvelopenet:latest --verbosity=info
```
Comment (andreyvelich) [inline test/scripts/v1alpha2/build-suggestion-nasenvelopenet.sh]: And here:
I think, here as well.
```
cp cmd/suggestion/v1alpha2/nasenvelopenet/Dockerfile .
gcloud builds submit . --tag=${REGISTRY}/${REPO_NAME}/v1alpha2/suggestion-nasenvelopenet:${VERSION} --project=${PROJECT}
gcloud container images add-tag --quiet ${REGISTRY}/${REPO_NAME}/v1alpha2/suggestion-nasenvelopenet:${VERSION} ${REGISTRY}/${REPO_NAME}/suggestion-nasenvelopenet:latest --verbosity=info

```

File: scripts/v1alpha1/build.sh (hunk 4 above)
Comment (gaocegege) [reply scripts/v1alpha1/build.sh]: Yeah, SGTM

**Comment by gaocegege** [comment]
Updated to sync the upstream master.

PTAL @johnugeorge @hougangliu

**Pull request merged**